### PR TITLE
#566: Adding tests for addresses

### PR DESCRIPTION
**Pull request by hani-ibrahim**
remaining work for #566

### CODE REVIEW

File: AtlasUI/AtlasUITests/AddressFormViewControllerTests.swift [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+import AtlasMockAPI
+
+@testable import AtlasUI
+@testable import AtlasSDK
+
+class AddressFormViewControllerTests: XCTestCase {
+
+    let window = UIWindow()
+
+    override func setUp() {
```
Comment (bonkey) [inline AtlasUI/AtlasUITests/AddressFormViewControllerTests.swift]: Could be `class` and `tearDown` as well.

File: AtlasUI/AtlasUITests/LoggedInAddressListActionHandlerTests.swift [hunk 2]
```diff
@@ -12,6 +12,7 @@ import AtlasMockAPI
 class LoggedInAddressListActionHandlerTests: XCTestCase {
 
     let delegate = AddressListTableDelegate(tableView: UITableView(), addresses: [], selectedAddress: nil, viewController: nil)
+    let window = UIWindow()
 
     override func setUp() {
```
Comment (bonkey) [inline AtlasUI/AtlasUITests/LoggedInAddressListActionHandlerTests.swift]: Could be `class` and `tearDown`, too.

File: AtlasUI/AtlasUITests/AddressFormViewControllerTests.swift (hunk 1 above)
Comment (bonkey) [inline AtlasUI/AtlasUITests/AddressFormViewControllerTests.swift]: Starts/stops server once per class containing tests, not for each of its tests.

Comment (hani-ibrahim) [inline AtlasUI/AtlasUITests/AddressFormViewControllerTests.swift]: 👍

**Comment by hani-ibrahim** [comment]
👍

**Comment by bonkey** [comment]
👍

**Comment by hani-ibrahim** [comment]
👍

**Comment by bonkey** [comment]
👍

**Pull request merged**